### PR TITLE
webview error handling 

### PR DESCRIPTION
**Pull request by AmirDoreh**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -38,7 +36,7 @@ with manual, [see more](https://github.com/react-native-community/react-native-w
 
 ## How to get `appId`, `appSecret` of instagram?
 
-> [Simple setup](https://developers.facebook.com/docs/instagram-basic-display-api/getting-started), You only need to complete the first 3 steps.
+> [Simple setup](https://developers.facebook.com/docs/instagram-basic-display-api/getting-started), you only need to complete step 3.
```
Comment (hungdev) [inline readme.md:39]: No, my mean is the first 3 steps: 1,2,3. It's not only the third step.

Comment (hungdev) [reply readme.md:39]: @AmirDoreh could you revert readme.md?

Comment (AmirDoreh) [reply readme.md:39]: sure

Comment (AmirDoreh) [reply readme.md]: readme has been updated @hungdev

Comment (hungdev) [reply readme.md]: @AmirDoreh and do not remove this tag: ## [Here is a video tutorial](https://www.youtube.com/watch?v=Xw-mCiyUMgE)

Comment (hungdev) [reply readme.md]: @AmirDoreh i would like to keep the tutorial in readme please don't remove it

**Review by hungdev — APPROVED** [review]

**Comment by hungdev** [comment]
Thank you so much <3

File: readme.md (hunk 1 above)
Comment (AmirDoreh) [reply readme.md]: done

Comment (hungdev) [reply readme.md]: I published new version. thank you.

**Pull request merged**